CHANGES.md

```diff
@@ -11,14 +11,27 @@ Rust image aims to be a pure-Rust implementation of various popular image format
 
 ### Unreleased
 
+- More convenient to use differently laid-out buffers for initialization and
+  results, but not operation directly, will be added in the future. The plan is
```

Comment:
This wording is somewhat awkward. I'm not entirely sure what "initialization and results, but not operation directly" means.